src/Elastic.Markdown/Slices/Layout/_TableOfContents.cshtml

```diff
@@ -1,7 +1,65 @@
+@using Elastic.Markdown.Helpers;
 @inherits RazorSlice<LayoutViewModel>
 <aside class="sidebar hidden lg:block order-3">
-	<nav id="toc-nav" class="sidebar-nav pl-4">
-		<div class="pt-6 pb-20">
+	<nav id="toc-nav" class="sidebar-nav">
+		<div id="page-version-dropdown" tabindex="1"
+		     class="mt-6 block group font-sans text-sm relative z-50">
+			<button
+				class="text-left border-1 rounded-2xl grid grid-cols-[1fr_auto] cursor-pointer font-semibold gap-1 hover:text-black pl-4 pr-1 py-1 group-open:border-b-1 border-grey-20">
+				<div>
+					<a
```

Comment:
Let's replace he `a` tag with a `span`. We don't need the clickable behaviour in this dropdown. Also, remove the hover effect etc.